Resources/Prototypes/DeltaV/Entities/Objects/Specific/Justice/gavel.yml

```
@@ -0,0 +1,18 @@
+- type: entity
+  parent: BaseItem
+  id: Gavel
+  name: gavel
+  description: A hardwood mallet made to keep order in the court.
+  components:
+    - type: Sprite
+      sprite: DeltaV/Objects/Specific/Justice/gavel.rsi
+      layers:
+        - state: icon
+    - type: MeleeWeapon
+      wideAnimationRotation: -90
+      damage:
+        types:
+          Blunt: 5
+    - type: Item
+      size: Small
+      sprite: DeltaV/Objects/Specific/Justice/gavel.rsi
```

A) Maybe this could be a steal objective?
B)
```suggestion
      sprite: DeltaV/Objects/Specific/Justice/gavel.rsi

```